.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  root: true,
+  env: {
+    es2020: true,
+    node: true,
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2020,
+  },
+  plugins: ["@typescript-eslint", "prettier"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier/@typescript-eslint",
+  ],
+  ignorePatterns: ["build/", "node_modules/", "!.prettierrc.js"],
```

Comment:
I think there is a double negation here because of the `!`: we don't want `.prettierrc.js` to be ignored, but no previous rules ignore it. I'd say we may as well also lint this file, so this pattern can be removed:
```suggestion
  ignorePatterns: ["build/", "node_modules/"],
```

Reply:
From what I understand, this is required because `.dotfiles` are ignored by default by ESLint. Without this `.prettierrc.js` does not get linted :cry:

Reply:
Does it make sense to do something like this then:
```suggestion
  ignorePatterns: ["build/", "node_modules/", "!.*"],
```
?
From some testing it seems to be working for `.prettierrc.js` and `.eslintrc.js` but not other non-js dotfiles.